fabtests/prov/efa/src/rdm_ep_rnr_retry.c

```diff
@@ -226,6 +226,13 @@ int main(int argc, char **argv)
 
 	hints->ep_attr->type = FI_EP_RDM;
 	hints->caps = FI_MSG;
+	/*
+	 * RNR error is generated from efa device, so disable shm
+	 * transfer by setting FI_REMOTE_COMM and unsetting FI_LOCAL_COMM
+	 * in order to make sure the test can be running on a single node.
```

Comment:
in order to make sure EFA device is being used.?

Reply:
Updated.